implementations/rust/ockam/ockam_command/src/identity/create.rs

```diff
@@ -71,13 +80,35 @@ impl CreateCommand {
 
             let vault = vault_state.get().await?;
 
-            let identity = opts
+            let identities_creation = opts
                 .state
                 .get_identities(vault)
                 .await?
-                .identities_creation()
-                .create_identity()
-                .await?;
+                .identities_creation();
+
+            let identity = match &self.key_id {
+                Some(key_id) => {
```

Comment:
A short comment here explaining what's going to happen would be nice. Something like `Create an identity using the provided KMS key`